pkg/evm/evm.go

```diff
@@ -264,21 +264,31 @@ func FundAddress(
 	return nil
 }
 
-func GetSignedTxToMethodWithWarpMessage(
+func GetTxToMethodWithWarpMessage(
 	client ethclient.Client,
+	generateRawTxOnly bool,
+	from common.Address,
 	privateKeyStr string,
 	warpMessage *avalancheWarp.Message,
 	contract common.Address,
 	callData []byte,
 	value *big.Int,
 ) (*types.Transaction, error) {
 	const defaultGasLimit = 2_000_000
```

Comment:
should we add a check here where either privateKeyStr != "" or from != "" and both can't be empty and both can't be not empty also?

Reply:
same for txtomethod